kafka-rest/src/main/java/io/confluent/kafkarest/ConsumerState.java

```diff
@@ -21,8 +21,7 @@
 import java.util.Map;
 import java.util.TreeMap;
 import java.util.Vector;
-import java.util.concurrent.locks.ReadWriteLock;
-import java.util.concurrent.locks.ReentrantReadWriteLock;
+import java.util.concurrent.locks.ReentrantLock;
```

Comment:
For the old consumer, the read/write lock is fine. The underlying `ConsumerIterator` class was thread-safe.